test/compound_and_array_test.mjs

```diff
@@ -1,51 +1,94 @@
 #!/usr/bin/env node
 
 import { strict as assert } from 'assert';
-import h5wasm from "../dist/node/hdf5_hl.js";
+import h5wasm from '../dist/node/hdf5_hl.js';
 
 async function compound_array_test() {
-
   await h5wasm.ready;
-  var f = new h5wasm.File("./test/array.h5", "r");
+  var f = new h5wasm.File('./test/array.h5', 'r');
```

Comment:
Just to be clear, when do you guys use single or double quotes? If this is an autoformatting issue, I suggest adding something like a `.editorconfig` for consistency, but looks like this is intentional?

Reply:
> Just to be clear, when do you guys use single or double quotes? 

We let prettier deal with this for us. Sorry about the reformating, I am so used to run it on save to format my code that I didn't pay much attention here...

I am happy to go with whatever style is preferred here but it would indeed be nice to set it in a `.editorconfig` or `.prettierrc`.